packages/app/src/contacts/components/contact-name/contact-name.component.tsx

```diff
@@ -0,0 +1,33 @@
+/**
```

Comment:
(not related to this file, just it is easier to have a thread)

Seems to me that if you decide to add `expand: true` option to `MessageSearcher` you would get better results.

Might be as scope of another task/PR + would be good to know if there are any drawbacks of this approach.

Example without `expand: true`
![image](https://user-images.githubusercontent.com/41268731/187692524-bb4c44c8-7d07-4af1-b38c-ab1487d1248b.png)

Example with `expand: false`
![image](https://user-images.githubusercontent.com/41268731/187692184-eba0d5db-3da6-47c6-b05e-4eb3e8a8f7eb.png)

Reply:
Looks promising, I'll try it in another PR